keyboards/mini49/mini49.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+#define LAYOUT_mini49( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, \
+    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, \
+    K40, K41, K42, K43,    K45,   K46, K47, K48, K49 \
+) \
+{ \
+    {K00, K01, K02, K03, K04, K05, K06, K07, K08, K09}, \
+    {K10, K11, K12, K13, K14, K15, K16, K17, K18, K19}, \
+    {K20, K21, K22, K23, K24, K25, K26, K27, K28, K29}, \
+    {K30, K31, K32, K33, K34, K35, K36, K37, K38, K39}, \
+    {K40, K41, K42, K43, ___, K45, K46, K47, K48, K49}  \
```

Comment:
You should look into migrating this information to `info.json`. (`qmk info -f json -kb mini49` may help.)

This is not currently a strict requirement for Vial submissions. If Vial continues to bring in changes from upstream QMK it will likely become one, and if that happens your board will break.
```suggestion
#define LAYOUT( \
    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, \
    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, \
    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, \
    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, \
    K40, K41, K42, K43,    K45,   K46, K47, K48, K49 \
) \
{ \
    {K00, K01, K02, K03, K04, K05, K06, K07, K08, K09}, \
    {K10, K11, K12, K13, K14, K15, K16, K17, K18, K19}, \
    {K20, K21, K22, K23, K24, K25, K26, K27, K28, K29}, \
    {K30, K31, K32, K33, K34, K35, K36, K37, K38, K39}, \
    {K40, K41, K42, K43, ___, K45, K46, K47, K48, K49}  \
```

Reply:
I'll check this out.